### PR TITLE
add vulkan-headers/1.2.198

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **vulkan-headers/1.2.198**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by madebr — DISMISSED** [review]

**Review by toge — APPROVED** [review]

File: recipes/vulkan-headers/all/conandata.yml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 sources:
+  "1.2.200":
+    url: "https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.2.200.zip"
+    sha256: "a3733e38f2f4243e4723fe85b4a80bdf85f02eafd154145e1708afa088a55b2a"
+  "1.2.198":
+    url: "https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.2.198.zip"
+    sha256: "0d0c54a6f96a0a1decdbb7270baf003a71b48c68b3516fed55882d9401e626c2"
```
Comment (SpaceIm) [inline recipes/vulkan-headers/all/conandata.yml:7]: tar.gz please

**Review by SpaceIm — APPROVED** [review]

**Comment by AndreyMlashkin** [comment]
@uilianries @SSE4, please approve

**Review by SSE4 — DISMISSED** [review]

**Comment by ericLemanissier** [comment]
is there a real need to make packages for so many versions ? Upstream project creates a new tag roughly once a week. In this PR we are creating a version 1.2.198 which is exactly identical to 1.2.198.0 (already packaged)

**Comment by SpaceIm** [comment]
It's for volk, recipe design prevents to easily depend on 1.2.198.0

**Review by ericLemanissier — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Comment by AndreyMlashkin** [comment]
@uilianries @SSE4 , please aprrove

**Review by danimtb — APPROVED** [review]

**Pull request merged**